tgui/packages/tgui/interfaces/PersonalCrafting.tsx

```diff
@@ -832,14 +832,14 @@ const RecipeContent = ({ item, craftable, busy, mode, diet }, context) => {
                   }
                 />
               )}
-              {item.nutriments > 0 && (
+              {item.complexity !== undefined && (
```

Comment:
```suggestion
              {!!item.complexity && (
```

Reply:
This would not allow complexity when it is 0.